src/js/releases.js

```diff
@@ -51,33 +53,78 @@ function buildLatestHTML(releasesJson) {
     // firstly, check if the platform name is recognised...
     if(thisPlatform) {
 
-      // secondly, check if the file has the expected file extension for that platform...
+      // secondly, search the assetArray for an installer for this platform, and create an HTML block for it
+      var thisInstallerBlock = "";
+      var thisInstallerExtension = getInstallerExt(thisPlatform); // get the file extension associated with this platform
+
+      assetArray.forEach(function(eachAsset2) {
+        var nameOfFile2 = (eachAsset2.name);
```

Comment:
eachAsset2 is a code smell :-)  What's the difference between eachAsset and eachAsset2?  And exactly what sort of asset is this?